libraries/botbuilder-stdlib/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "botbuilder-stdlib",
+  "author": "Microsoft Corp.",
+  "description": "BotBuilder shared libraries, intenal only",
+  "version": "4.1.6",
+  "preview": true,
+  "license": "MIT",
+  "keywords": [
+    "botbuilder",
+    "botframework",
+    "bots",
+    "chatbots"
+  ],
```

`"keywords"` are not necessary since this is an "internal" library, we don't necessarily _want_ people to find this on `npm` 😃 

```suggestion
  "keywords": [],
```